srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -17,6 +33,146 @@
+        }
+
+        public static ModelTermSummary createTermSummary( ModelTerm modelTerm ) 
+        {
+            String dispName="";
+            String type="";
+            
+            if( modelTerm instanceof ConstantTerm ) 
+            {
+                dispName = "Constant"; 
+                type = "Constant"; // or N/A or blank
+            }
+            else if( modelTerm instanceof ProcedureTerm )
+            {
+                type = "Multiplier"; // ???
+                dispName = "Procedure (RVU Multiplier)"; 
+            }
+            else if( modelTerm instanceof BooleanTerm )
+            {
+                type = "Boolean";
+                BooleanTerm boolTerm = (BooleanTerm)modelTerm;                
+                dispName = boolTerm.getVariable().getDisplayName();
+            }            
+            else if( modelTerm instanceof NumericalTerm )
+            {
+                type = "Numerical";
+                NumericalTerm numTerm = (NumericalTerm)modelTerm;
+                dispName = numTerm.getVariable().getDisplayName();                                     
+            }
+            else if( modelTerm instanceof DiscreteTerm )
+            {
+                DiscreteTerm discreteTerm = (DiscreteTerm)modelTerm;                                
+                MultiSelectOption opt = discreteTerm.getOption();
+                DiscreteVariable var = discreteTerm.getVariable();
+                
+                if( var instanceof DiscreteNumericalVariable ) 
+                {
+                    type = "Discrete Numeric";
+                    dispName = var.getDisplayName()+ " = " + opt.getValue();
+                }
+                else if( var instanceof MultiSelectVariable ) 
+                {
+                    type = "Discrete";
+
+                    MultiSelectVariable msv = (MultiSelectVariable)var;
+                    msv.getOptions();
+                    dispName = var.getDisplayName()+ " = " + opt.getValue() ;
+                }
+            }            
+            else if( modelTerm instanceof DerivedTerm )
+            {
+                type = "Rule";
+                dispName = "Rule: "+((DerivedTerm)modelTerm).getRule().getId();
+            }
+            else
+            {
+                // or throw an error?
+                dispName = "Unrecognized Term: "+modelTerm.getClass().getName();    
+                type = "";
+            }
+            
+            // Use new Type column to display the Type.
+            // Can still do this if that's what they want.
+            // dispName = dispName + " ("+type+")";
+            
+            return new ModelTermSummary( modelTerm, dispName, type );
+        }
+
+        // a string used to make sorting the terms easier. 
+        // ConstantTerms first, then Rules and then by display name 
+        // with discrete options ordered by index 
+        private String getSortString() 
```

Comment:
You can make these comments into a javadoc comment to clean them up and add some information, even though the method is private.